http-common/src/server.rs

```diff
@@ -96,24 +96,27 @@ macro_rules! make_service {
                                 return Box::pin(async move {
                                     let response = match method {
                                         http::Method::DELETE => {
-                                            let body = {
+                                            let body = match tokio::time::timeout(HYPER_REQUEST_TIMEOUT, hyper::body::to_bytes(body)).await {
+                                                Ok(result) => match result {
```

Comment:
You could un-nest the `match` to make it slightly more compact:

```rust
let body = match tokio::time::timeout(HYPER_REQUEST_TIMEOUT, hyper::body::to_bytes(body)).await {
    Ok(Ok(body)) => body,
    Ok(Err(err)) => return Ok((http_common::server::Error { ... }).to_http_response(),
    Err(timeout_err) => return Ok((http_common::server::Error { ... }).to_http_response(),
};
```

Reply:
Fixed